packages/core/src/PlayerContextProvider.js

```diff
@@ -340,11 +345,18 @@ export class PlayerContextProvider extends Component {
     }
 
     // if not, then load the first track in the new playlist, and pause.
+    const currentTime = newPlaylist[0].startingTime || 0;
     return {
       ...baseNewState,
-      ...getGoToTrackState({ prevState, index: 0, shouldPlay: false }),
+      ...getGoToTrackState({
+        prevState,
+        currentTime,
+        index: 0,
+        shouldPlay: false
+      }),
       mediaCannotPlay: false,
-      awaitingPlayAfterTrackLoad: false
+      awaitingPlayAfterTrackLoad: false,
+      trackLoading: true
```

Comment:
Seems like `trackLoading: true` is already there from the call to `getGoToTrackState`. Same for `awaitingPlayAfterTrackLoad` actually.

Reply:
Not unless I pass `shouldForceLoad: true`, since trackLoading is set to true if the activeTrackIndex changes, and I can modify the playlist to remove the first track, and activeTrackIndex wouldn't change.

Reply:
Good point.. maybe we should just set force load to true?

Reply:
Sure, that makes sense, it will lead to the same behaviour in componentDidUpdate.